site/viamguides/src/rgbled/rgbled.md

```diff
@@ -0,0 +1,274 @@
+1. Connect your Pi to Wi-Fi so that you can run `viam-server` wirelessly. Check **Configure wireless LAN** and enter your wireless network credentials. SSID (short for Service Set Identifier) is your Wi-Fi network name, and password is the network password. Change the section `Wireless LAN country` to where your router is currently being operated.
+   ![raspberry pi hostname username and password](assets/generalSettings.png)
+1. Select the **SERVICES** tab, check **Enable SSH**, and select **Use password authentication**.
+   ![raspberry pi enable SSH](assets/sshSettings.png)
+   > aside negative
+   > Be sure that you remember the `hostname` and `username` you set, as you will need this when you SSH into your Pi.
+1. **Save** your updates, and confirm `YES` to apply OS customization settings. Confirm `YES` to erase data on the USB flash drive. You may also be prompted by your operating system to enter an administrator password. After granting permissions to the Imager, it will begin writing and then verifying the Linux installation to the USB flash drive.
+1. Remove the USB flash drive from your computer when the installation is complete.
+
+### Connect with SSH
+
+1. Place the USB flash drive into your Raspberry Pi and boot the Pi by plugging it in to an outlet. A red LED will turn on to indicate that the Pi is connected to power.
+   > aside negative
+   > Make sure you are using a 5V 3A USB-C power supply to power your Raspberry Pi 4. Using a power supply with inadequate amperage can lead to instability, throttling, or unexpected behavior. Additionally, while USB boot is enabled by default on newer Raspberry Pi 4 models, older versions may require a firmware update to enable it. Refer to the [Raspberry Pi documentation](https://www.raspberrypi.com/documentation/computers/raspberry-pi.html) for detailed setup instructions and compatibility.
+1. Once the Pi is started, connect to it with SSH. From a command line terminal window, enter the following command. The text in <> should be replaced (including the < and > symbols themselves) with the user and hostname you configured when you set up your Pi.
+   ```bash
+   ssh <USERNAME>@<HOSTNAME>.local
+   ```
+1. If you are prompted “Are you sure you want to continue connecting?”, type “yes” and hit enter. Then, enter the password for your username. You should be greeted by a login message and a command prompt.
+   ![raspberry pi SSH login](assets/sshLogin.png)
+1. Update your Raspberry Pi to ensure all the latest packages are installed
+   ```bash
+   sudo apt update
+   sudo apt upgrade
+   ```
+
+<!-- ------------------------ -->
+
+## Set up the hardware
+
+Duration: 2
+
+### Add your RGB LED
+
+1. **Connect the RGB LED**: The LED can be controlled via a GPIO pin on the Raspberry Pi. Refer to the following wiring diagram to connect the Raspberry Pi to the RGB LED, using the breadboard and also resistors to control the flow of electricity.
+
+![wiring diagram](assets/rgbledWiring.png)
```

Comment:
This diagram shows a Raspberry Pi 5 even though the Raspberry Pi 4 is used in the prerequisites. There's no difference in the physical pins between the two boards but some folks could be confused by the discrepancy, so we should add an aside below to image to assuage any concerns.